### ISSUE TITLE
Checkout of existing topic issue.

### ISSUE THREAD

**Issue by wilx** [magit/magit-topgit#1]
I have just experienced an odd issue when trying to check out existing topic:

```
  1 git … checkout t/1.1.x/sockets-use-socket-holder  [PATCH]
error: pathspec 't/1.1.x/sockets-use-socket-holder  [PATCH]' did not match any file(s) known to git.
```

As you can see, it basically tried to use more than just the selected branch from `magit-status` buffer instead of just the branch:

```
        t/1.1.x/sockets-use-socket-holder   [PATCH] t/1.1.x/sockets-use-socket-holder
```

My `magit-version` prints "Magit 20151028.1649, Git 2.5.0, Emacs 24.5.1".

**Comment by tarsius** [comment]
The problem would likely be here:

``` lisp
(defconst magit-topgit-topic-re "^\\(.\\{7\\}\\)\t\\([^ ]+\\) +\\(.*\\)")

(defun magit-topgit-read-topic (prompt)
  (magit-completing-read
   prompt (mapcar (lambda (line)
                    (string-match magit-topgit-topic-re line)
                    (match-string 2 line))
                  (process-lines magit-topgit-executable "summary"))
   nil t))
```

I don't have topgit installed, so what's the output of `topgit summary`?

**Comment by tarsius** [comment]
> I have just experienced an odd issue when trying to check out existing topic:

Using `magit-topgit-checkout`?

**Comment by wilx** [comment]
Oops. Sorry, I somehow missed your comment 16 days ago.

> I don't have topgit installed, so what's the output of `topgit summary`?

I have finished the stuff and then manually removed the branches, etc., after I merged the changes to master. I have tried reproduce this with different names but so far unsuccessfully. I will close this issue for now. I will reopen when/if I see another instance of this issue.

> Using `magit-topgit-checkout`?

I was using the Magit UI. But yes, that seems to be the same thing.
